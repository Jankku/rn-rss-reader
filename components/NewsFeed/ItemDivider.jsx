import { memo } from 'react';
import { useTheme } from '@react-navigation/native';
import { View } from 'react-native';

function ItemDivider() {
  const { colors } = useTheme();

  return (
    <View
      style={{
        height: 1,
        backgroundColor: colors.border,
      }}
    ></View>
  );
}

export default memo(ItemDivider);
