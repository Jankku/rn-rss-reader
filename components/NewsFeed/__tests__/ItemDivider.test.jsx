import ItemDivider from '../ItemDivider';
import { render } from '@testing-library/react-native';

describe('Test ItemDivider', () => {
  test('Should match snapshot', () => {
    const { toJSON } = render(<ItemDivider />);
    expect(toJSON()).toMatchSnapshot();
  });
});
