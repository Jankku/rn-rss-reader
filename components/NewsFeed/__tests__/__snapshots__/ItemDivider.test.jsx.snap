// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ItemDivider Should match snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(216, 216, 216)",
      "height": 1,
    }
  }
/>
`;
